app/helpers/user_helper.rb

```diff
@@ -0,0 +1,2 @@
+module UserHelper
+end
```

Comment:
We shouldn't keep empty files (seems like we don't use those modules even for namespacing)